script/build.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+
+##
+# Run the application and content builds. This script builds or fetches the
+# pre-built application assets based on the --asset-source option.
+###
+
+# Default to local
+assetSource="local"
+
+# Save the arguments to this script for later; they get drained in the following for loop
+args="$*"
```

Comment:
I think I saw a better way to do this the other day, but didn't write it down and goodness knows I won't be able to find it again. Stinking bashisms.